mycrt-backend/src/database/user_database.py

```diff
@@ -1,12 +1,14 @@
 from sqlalchemy import create_engine
 from sqlalchemy.orm import scoped_session, sessionmaker
 from flask_security import SQLAlchemySessionUserDatastore
-from flask_sqlalchemy import *
 
 from .base import Base
 from .user import User, Role, RolesUsers
+import src.database.models
 
-class UserRepository:
+from .db_config import db_string
+
+class MycrtDatabase:
```

Comment:
nitpicking but can we change this to MyCRTDatabase?

Reply:
sure